casepro/cases/models.py

```diff
@@ -492,7 +492,8 @@ def as_json(self):
         return {
             'id': self.pk,
             'action': self.action,
-            'created_by': self.created_by.as_json(full=False),
+            'created_by': self.created_by.as_json(full=False)
+            if self.created_by else None,
```

Comment:
I think this can fit on one line, since we have 120 chars to work with in casepro